bundle/manifests/cert-manager-operator.clusterserviceversion.yaml

```diff
@@ -711,10 +711,10 @@ spec:
   provider:
     name: Red Hat
   relatedImages:
-  - image: quay.io/jetstack/cert-manager-webhook:v1.9.1
+  - image: quay.io/jetstack/cert-manager-webhook:v1.10.2
     name: cert-manager-webhook
-  - image: quay.io/jetstack/cert-manager-cainjector:v1.9.1
+  - image: quay.io/jetstack/cert-manager-cainjector:v1.10.2
     name: cert-manager-ca-injector
-  - image: quay.io/jetstack/cert-manager-controller:v1.9.1
+  - image: quay.io/jetstack/cert-manager-controller:v1.10.2
     name: cert-manager-controller
-  version: 0.0.1
+  version: 1.10.2
```

Comment:
Change this back to `0.0.1` sine the bundle version was reverted as well.

Reply:
Done.